python-modules/cis_change_service/cis_change_service/__init__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+"""Flaks application for publishing changes."""
+
+__author__ = """Andrew Krug"""
+__email__ = 'akrug@mozilla.com'
+__version__ = '0.0.1'
+
+import os
+
+from everett.manager import ConfigManager
+from everett.manager import ConfigIniEnv
+from everett.manager import ConfigOSEnv
+
+
+def get_config():
```

Comment:
i think its missing `__all__ = [get_config]

Reply:
I need to go back and do this everywhere...